app/assets/stylesheets/wizard.scss

```diff
@@ -406,13 +406,6 @@ body.wizard {
   }
 
   &__button.jump-in {
-    background-color: var(--tertiary);
-    color: var(--secondary);
-    margin-left: 1em;
-    @include breakpoint("mobile-extra-large") {
-      order: 1;
-      margin-left: 0;
-    }
     &:hover {
       background-color: var(--primary-300);
     }
```

Comment:
I don't actually think this is intentional, but I kept it here.

Previously, the "Jump In" button did not have the `.primary` CSS class, but it very much is and wants to be the primary action, and duplicated a bunch of styles from `.primary`. Now, I just added the primary class to the "Jump In" button and removed the redundant rules. This is the only difference left and it makes the hover shade a bit different than the "Next" button. To my eyes it looked off and I think we probably just want to remove this, but I left it as is for now.